### DISCUSSION TITLE
Sessions

### DISCUSSION THREAD

**Discussion by rebop** [onemen/TabMixPlus#229] [Q&A]
Are you ever going to implement sessions again? If not, I will forget browsing these pages. If so, I will be happy to beta test for you. There are few, if any, great session managers right now.

**Comment by onemen** [comment]
It will require a lot of my free time, and Tan mix don't have enough users anymore to justify it

**Reply by 117649** [reply]
I think what it is useful to user now is the UI of the TMP session manager.
How about hook that up with the back end of the built-in one?

**Reply by onemen** [reply]
Tab MIx UI for Session manager is 12 years old. Any implementation of Session manager in Tab mix will require complete rewrite

**Reply by 117649** [reply]
Are those code still in the repo?

**Reply by onemen** [reply]
Yes,
